### ISSUE TITLE
Strange characters after server start

### ISSUE THREAD

**Issue by victorbuttner** [dasisdormax/csgo-multiserver#6]
Hi,

I'm having an strange behaviour on my server. After start the server on the bash console I got some strange characters like `62;c`
Did you know what it is it ?
```
[detached (from session csgo@)]
**SUCCESS:** 
    **Base Installation** started successfully!
    
    Use **csgo-server @ console** to enter the game's console.

ubuntu@myserver:~$ 62;c^C
```

**Comment by dasisdormax** [comment]
Hi,

I've seen this occasionally before, but don't know the exact cause. To me, this looks like an escape sequence that isn't output or parsed correctly. This may be a conflict between tmux and this script, but may as well be caused by your terminal emulator.

Currently, I don't have the time to look more deeply into it, so you may have to live with this problem for now

**Comment by hmmalafaia** [comment]
Hi, @dasisdormax 

Someone can help me with that?
Looks like server started successfully, not =\
![error](https://user-images.githubusercontent.com/14101961/107990484-52911100-6fb3-11eb-8972-503571fb6606.PNG)

**Comment by dasisdormax** [comment]
Hello @hmmalafaia

This looks like an incompatible tmux version to me, can you tell me your operating system and tmux version? (tmux -V)

**Comment by dasisdormax** [comment]
The original issue might be fixed by #11
